Add auto mtls in annoucing doc for 1.4. (#5798)
istio/istio.io: content/en/news/releases/1.4.x/announcing-1.4/_index.md

focuses on simplification and flexibility. This will also replace the old
[`v1alpha1` RBAC policy](/docs/reference/config/security/istio.rbac.v1alpha1/).

## Automatic mutual TLS

We added [automatic mutual TLS support](/docs/tasks/security/authentication/auto-mtls/). It allows

> you to adopt mutual TLS without configuring destination rule. Istio automatically configures

Reviewer: ```suggestion
you to adopt mutual TLS without needing to configure destination rules. Istio automatically programs
```

> client sidecar proxy to send mutual TLS to server endpoints those are able to accept mutual TLS.

Reviewer: ```suggestion
client sidecar proxies to send mutual TLS to server endpoints that are able to accept mutual TLS.
```

> Currently this feature requires to explicitly enabled, and we plan to enable it by default in

Reviewer: ```suggestion
Currently this feature must be explicitly enabled, but we plan to enable it by default in a
```
